sdk/eventhub/azure-eventhub/samples/README.md

```diff
@@ -0,0 +1,95 @@
+    - Receive events and do checkpoint every fixed amount of events (e.g. checkpoint every 20 events) using blob checkpoint store
+
+- [recv_with_checkpoint_by_time_interval.py](./sync_samples/recv_with_checkpoint_by_time_interval.py) ([async version](./async_samples/recv_with_checkpoint_by_time_interval_async.py)) - Examples to receive events and do checkpoint by time interval using blob checkpoint store:
+    - Receive events and do checkpoint every fixed time interval (e.g. checkpoint every 20 seconds) using blob checkpoint store
+
+- [recv_for_period.py](./sync_samples/recv_for_period.py) ([async version](./async_samples/recv_for_period_async.py)) - Examples to receive events for a period of time:
+    - Receive events for a period of time
+
+
+- [client_identity_authentication.py](./sync_samples/client_identity_authentication.py) ([async version](./async_samples/client_identity_authentication_async.py)) - Examples for authentication by Azure Active Directory:
+    - Authenticating and creating the client utilizing the `azure.identity` library
+
+
+- [proxy.py](./sync_samples/proxy.py) ([async version](./async_samples/proxy_async.py)) - Examples to send and receive events behind a proxy:
+    - Send and receive events behind a proxy
+
+## Prerequisites
+- Python 2.7, 3.5 or later.
+- **Microsoft Azure Subscription:**  To use Azure services, including Azure Event Hubs, you'll need a subscription.
+If you do not have an existing Azure account, you may sign up for a free trial or use your MSDN subscriber benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+
+- **Event Hubs namespace with an Event Hub:** To interact with Azure Event Hubs, you'll also need to have a namespace and Event Hub  available.
+If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide
+for [creating an Event Hub using the Azure portal](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
+There, you can also find detailed instructions for using the Azure CLI, Azure PowerShell, or Azure Resource Manager (ARM) templates to create an Event Hub.
+
+- **Azure Storage Account (Optional)**: To run receiving samples with blob checkpoint store for persist checkpoint, you need to [create an Azure Storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal) and a [Blob Container](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal#create-a-container).
+
+## Setup
+
+1. Install the Azure Event Hubs client library for Python with [pip](https://pypi.org/project/pip/):
+```bash
+pip install azure-eventhub
+```
+To run receiving samples with blob checkpoint store for persist checkpoint, please install the corresponding checkpoint library:
+```bash
+pip install azure-eventhub-checkpointstoreblob  # sync version
+pip install azure-eventhub-checkpointstoreblob-aio  # async version
```

Comment:
What other libs are required for some samples. For instance azure-identity is required for some sample.

Reply:
Good catch!

Reply:
good catch!

Reply:
just added instruction for azure-identity, there're no other libraries needed